NetSSL_Win/src/Context.cpp

```diff
@@ -91,7 +91,11 @@ void Context::init()
 	_hMemCertStore = CertOpenStore(
 					CERT_STORE_PROV_MEMORY,   // The memory provider type
 					0,                        // The encoding type is not needed
-					NULL,                     // Use the default provider
+	#ifdef _MSC_VER                           // Use the default provider
+					NULL,
+	#else
+					0,
+	#endif
```

Comment:
please just use 0 here, but why is NULL not good here and is in the last argument?

Reply:
Because MinGW required number here